.github/workflows/new_target_network_notification.yml

```diff
@@ -0,0 +1,60 @@
+name: New Target network Slack Notification
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - "deployment"
+      - "DEV-nojira-test-merge-workflow"
```

Comment:
need to fix the branch name

I think the event should be on merge

Reply:
Branch name is added for testing, as I can't test this script on merging to develop

There is no action in GitHub as "merged", so the only way is to check when PR is closed if it has the event "merged"

Reply:
renamed "deployment" to "develop"
"DEV-nojira..." will be removed after the test